src/constant/chainConfig/localhost.ts

```diff
@@ -0,0 +1,19 @@
+import { ChainConfig } from "@/constant/chainConfig/types"
+
+export const localhostConfig: ChainConfig = {
+  registryAddress: "0x03605C3A3dAf860774448df807742c0d0e49460C",
```

Comment:
Does this match the arbitrum config? If so, you should just spread that in and overwrite what is necessary:

```ts
export const localhostConfig: ChainConfig = {
  ...arbitrumConfig,
  overrides: "go here",
}
```